firestore/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java

```diff
@@ -38,10 +38,12 @@
 /** Snippets to support firestore querying data documentation. */
 class QueryDataSnippets {
 
-  private final Firestore db;
+  private static final String COLLECTION_REFERENCE_NAME = "cities";
```

Comment:
In normal application code this would likely be a desired refactoring, but due to the fact that each of the users of this new variable are individual code snippets that are read on their own we prefer to keep the value inline.